templates/cluster-template.yaml

```diff
@@ -130,4 +136,6 @@ spec:
       joinConfiguration:
         nodeRegistration:
           kubeletExtraArgs:
+            cloud-provider: external
```

Comment:
By setting cloud-provider to external I got my pods stuck unschedulable.

```
Events:
  Type     Reason            Age                 From               Message
  ----     ------            ----                ----               -------
  Warning  FailedScheduling  21s (x13 over 14m)  default-scheduler  0/3 nodes are available: 1 node(s) had taint {node-role.kubernetes.io/master: }, that the pod didn't tolerate, 2 node(s) had taint {node.cloudprovider.kubernetes.io/uninitialized: true}, that the pod didn't tolerate.
```

Reply:
yes, this value should not be set unless we have cloud-provider-ibmcloud ready then use that cloud provider to run controller 
https://github.com/kubernetes/cloud-provider has further info

Reply:
So, there is no `cloud-provider-ibmcloud` Ready for use?
I'm asking that because I've tried to create a Service of type LoadBalancer but it got stuck due to the lack of a cloud-provider set on the cluster created by the capi provider ibm.

Reply:
2 things 
you need LB, then you can go ahead in capi-ibmcloud to create it through API exposed by IBM cloud
but if you create a service of type LB, check how openstack did https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md, they rely on external cloud provider now , but I think legacy way before cloud provider method should have simliar way 

but the question is you set `external` here which tells k8s to rely on external cloud provider while as far as I can tell it's existing but not open sourced yet .. so here's the gap I guess